shared/constants/chat.js

```diff
@@ -164,6 +164,26 @@ export type ChatSecuredHeaderMessage = {
   key: MessageKey,
 }
 
+export type SystemMessage = {
+  type: 'System',
```

Comment:
This seems like a new type. Copied definition over from TextMessage. @buoyad were you the one that added this? Does it look right?

Reply:
i think a bunch of these fields should be removed. there's no deviceName, type, you, failure, revoked, editedcount, mentions, channelmentions etc

Reply:
Done.